sbin/build.sh

```diff
@@ -978,7 +979,7 @@ generateSBoM() {
   local fullVer=$(cat "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[TARGET_DIR]}/metadata/productVersion.txt")
   local fullVerOutput=$(cat "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[TARGET_DIR]}/metadata/productVersionOutput.txt")
 
-  # Create initial SBOM json
+  # Create initial SBOM json 
```

Comment:
Extra space has appeared here
```suggestion
  # Create initial SBOM json
```